apps/admin/src/components/main/FormList/FormList.tsx

```diff
@@ -0,0 +1,143 @@
+import ListHeader from '@/components/common/ListHeader/ListHeader';
+import ListItem from '@/components/common/ListItem/ListItem';
+import { Column, Row, Text } from '@maru/ui';
+import { styled } from 'styled-components';
+
+const FORM_DATA: {
+    id: number;
+    name: string;
+    birthday: string;
+    graduationType: 'EXPECTED' | 'GRADUATED' | 'QUALIFICATION_EXAMINATION';
+    school: string;
+    status: '최종 제출됨' | '반려됨';
+    type:
+        | 'REGULAR'
+        | 'SPECIAL'
+        | 'MEISTER_TALENT'
+        | 'SPECIAL_ADMISSION'
+        | 'NATIONAL_BASIC_LIVING'
+        | 'NEAR_POVERTY'
+        | 'NATIONAL_VETERANS'
+        | 'ONE_PARENT'
+        | 'FROM_NORTH_KOREA'
+        | 'MULTICULTURAL'
+        | 'TEEN_HOUSEHOLDER'
+        | 'MULTI_CHILDREN'
+        | 'FARMING_AND_FISHING';
```

Comment:
```suggestion
    type: FormType;
```

아래 정의해놨으니 사용하는것이 좋을것같아요